tools/ovc/openvino/tools/ovc/moc_frontend/pytorch_frontend_utils.py

```diff
@@ -8,6 +8,14 @@
 # pylint: disable=no-name-in-module,import-error
 from openvino.runtime import Tensor, PartialShape
 from openvino.tools.ovc.error import Error
+from openvino.frontend.pytorch.module_extension import ModuleExtension
+
+
+def extract_module_extensions(args):
+    extensions = args.get('extension', [])
+    if not isinstance(extensions, (list, tuple)):
+        extensions = [extensions]
+    return {extension.module: extension for extension in extensions if isinstance(extension, ModuleExtension)}
 
 
 def get_pytorch_decoder(model, example_inputs, args):
```

Comment:
`TorchScriptPythonDecoder` is imported below under try-except above and inside a function, not in global space unlike `ModuleExtension`.